### ISSUE TITLE
Add ox_lib Text ui

### ISSUE THREAD

**Issue by ItssJxstnDe** [greenghoost/GG-Drivingschool#1]
So No "interact" IS really needed

**Comment by greenghoost** [comment]
yes I could but my intension was to make it on interact actually.

**Comment by ItssJxstnDe** [comment]
So can you do pls?

**Comment by greenghoost** [comment]
> So can you do pls?

yes I can. just pushed new update today, will add in the next update soon. keep an eye to my discord
